sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/BlobHelper.java

```diff
@@ -0,0 +1,70 @@
+package com.azure.storage.blob.implementation.util;
+
+import com.azure.storage.blob.BlobServiceVersion;
+import com.azure.storage.blob.implementation.AzureBlobStorageImpl;
+import com.azure.storage.blob.models.CpkInfo;
+import com.azure.storage.blob.specialized.BlobAsyncClientBase;
+
+public class BlobHelper {
+
+    protected BlobHelper() {
```

Comment:
make this constructor `private` since all the methods in the class are `static`.